libraries/AP_Vehicle/AP_Vehicle.h

```diff
@@ -176,6 +176,8 @@ class AP_Vehicle : public AP_HAL::HAL::Callbacks {
     virtual bool start_takeoff(const float alt) { return false; }
     // Method to control vehicle position for use by external control
     virtual bool set_target_location(const Location& target_loc) { return false; }
+    // Get target location for use by external control
+    virtual bool get_target_location(Location& target_loc) { return false; }
```

Comment:
Every function called by DDS that was only enabled by scripting should always be moved into this block.
Otherwise, if you disable scripting and enable DDS, you can't compile.